store/v2/root/store.go

```diff
@@ -238,8 +238,7 @@ func (s *Store) LoadVersion(version uint64) error {
 // NOTE: It cannot be called while the store is migrating.
 func (s *Store) LoadVersionAndUpgrade(version uint64, upgrades *corestore.StoreUpgrades) error {
 	if s.telemetry != nil {
-		now := time.Now()
-		defer s.telemetry.MeasureSince(now, "root_store", "load_version_and_upgrade")
+		defer s.telemetry.MeasureSince(time.Now(), "root_store", "load_version_and_upgrade")
```

Comment:
```suggestion
		defer s.telemetry.MeasureSince(telemetry.Now(), "root_store", "load_version_and_upgrade")
```

this is done to avoid calling time when metrics are not enabled